python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -222,13 +222,14 @@ def fit(self,
                 You can also provide custom metrics by passing in a custom training_operator_cls
                 when creating the Estimator.
         """
+        shard_size = OrcaContext._shard_size if OrcaContext._shard_size else batch_size
```

Comment:
batch_size should override shard_size

Reply:
Should not?
batch_size will always be used as the default shard size.
But if users specify OrcaContext._shard_size, they wish to use this value and it should override batch size? Isn't this the logic?

Reply:
The user should not know whether we are using XShards or not inside Estimator; Estimator should manage its own XShards implementation itself.

Reply:
So you mean not use OrcaContext._shard_size, but always use batch_size and this value is not configurable?

Reply:
I think it's reasonable that the user should always configure batch_size when using Estimator.

Reply:
Yes, user would configure batch_size; and using batch_size to convert to xhards will always work.
So directly throw OrcaContext._shard_size here?

Reply:
> Yes, user would configure batch_size; and using batch_size to convert to xhards will always work. So directly throw OrcaContext._shard_size here?

I think you may ignore OrcaContext._shard_size here